contracts/access/IAccessControl.sol

```diff
@@ -30,8 +30,7 @@ interface IAccessControl {
     /**
      * @dev Emitted when `account` is granted `role`.
      *
-     * `sender` is the account that originated the contract call, an admin role
-     * bearer except when using {AccessControl-_setupRole}.
+     * `sender` is the account that originated the contract call i.e. the admin role bearer.
```

Comment:
When the role is granted using the internal `_grantRole` (for example during construction, or through any other process), then the `msg.sender` may not bear the admin rights for the role that is granted.

Reply:
Oh right, my bad, so it should be 
```
`sender` is the account that originated the contract call i.e. the admin role bearer
except when using {AccessControl-_grantRole}.
```

Reply:
```suggestion
     * `sender` is the account that originated the contract call. This account bears the admin role (for the granted 
     * role), expect in cases where the role was granted using the internal {AccessControl-_grantRole}.
```

Reply:
If that's the case,  should we update the description for the event `RoleRevoked` as well then?

```
/**
 * @dev Emitted when `account` is revoked `role`.
 *
 * `sender` is the account that originated the contract call:
 *   - if using `revokeRole`, it bears the admin role (for the granted role)
 *   - if using `renounceRole`, it is the role bearer (i.e. `account`)
 */
event RoleRevoked(bytes32 indexed role, address indexed account, address indexed sender);
```

Reply:
I think the current description of `RoleRevoked` is fine. Given there are already contracts out there that may grant roles without emitting `RoleGranted`, it justifies clarifying only for that case.

Reply:
I think the current description of `RoleRevoked` is fine. Given there are already contracts out there that may grant roles without emitting `RoleGranted`, it justifies clarifying only for that case.